extras/spoof.md

```diff
@@ -47,6 +47,38 @@ With this, navigate towards `Root -> DeviceProperties -> Add` and add your PCI r
 
 ![](../images/extras/spoof-md/config-gpu.png)
 
+For disabling NVIDIA Kepler you can use these device prperties:
+
+```
+<key>DeviceProperties</key>
+	<dict>
+    <key>PciRoot(0x0)/Pci(0x7,0x0)/Pci(0x0,0x0)/Pci(0x8,0x0)/Pci(0x0,0x0)</key>
+			<dict>
+				<key>AAPL,ndrv-dev</key>
+				<false/>
+				<key>agdpmod</key>
+				<data>
+				cGlrZXJhAA==
+				</data>
+				<key>rebuild-device-tree</key>
```

Comment:
Why?

Reply:
cGlrZXJhAA== is pikera. It might not be needed here
wegtree=1 boot argument (rebuild-device-tree property) to force device renaming on Apple FW. So this is also specific for Apple firmware and may be omitted for general application.

Reply:
Regarding pikera, I don't see the point of having it. And with wegtree, which macs have this property defined? I looked through a few firmware dumps quick and no Kepler macs I saw had the property